src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.page {
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    width: $calendar-width;
+    padding: 0;
+    margin: auto;
+    gap: $gap;
+
+    &__day {
+        box-sizing: border-box;
+        width: $day-size;
+        height: $day-size;
+        background-color: $day-color;
```

Comment:
The variable `$day-color` is used here but is not defined in this file. Ensure that it is defined or imported from another file.